sbot/__init__.py

```diff
@@ -14,6 +14,8 @@
 COAST = MotorSpecialState.COAST
 BRAKE = MotorSpecialState.BRAKE
 
+AnaloguePin = SBArduinoBoard.AnaloguePin
```

Comment:
This feels like a strange change?

Reply:
https://github.com/j5api/j5/commit/30d8784a3e692a028823079f9fbe67eaf8e11474